touchforms/backend/test_files/xforms/repeat.xml

```diff
@@ -0,0 +1,229 @@
+	</h:head>
+	<h:body>
+		<trigger ref="/data/new_intro" appearance="minimal">
+			<label ref="jr:itext('new_intro-label')"/>
+		</trigger>
+		<group>
+			<label ref="jr:itext('repeatable-label')"/>
+			<repeat nodeset="/data/repeatable">
+				<input ref="/data/repeatable/value">
+					<label ref="jr:itext('value-label')"/>
+				</input>
+				<select1 ref="/data/repeatable/choose_option">
+					<label ref="jr:itext('repeatable/choose_option-label')"/>
+					<item>
+						<label ref="jr:itext('question5-one-label')"/>
+						<value>one</value>
+					</item>
+					<item>
+						<label ref="jr:itext('question5-two-label')"/>
+						<value>two</value>
+					</item>
+				</select1>
+			</repeat>
+		</group>
+		<trigger ref="/data/toofew">
+			<label ref="jr:itext('toofew-label')"/>
+		</trigger>
+		<trigger ref="/data/repeat_total">
+			<label ref="jr:itext('repeat_total-label')"/>
+		</trigger>
+		<group>
+			<label ref="jr:itext('outer_repeat-label')"/>
+			<repeat jr:count="/data/outer_count" jr:noAddRemove="true()" nodeset="/data/outer_repeat">
+				<trigger ref="/data/outer_repeat/repeated_group">
+					<label ref="jr:itext('question28-label')"/>
+				</trigger>
+				<group>
+					<label ref="jr:itext('inner_repeat-label')"/>
+					<repeat jr:count="/data/inner_count" jr:noAddRemove="true()" nodeset="/data/outer_repeat/inner_repeat">
+						<input ref="/data/outer_repeat/inner_repeat/inner_question">
+							<label ref="jr:itext('inner_question-label')"/>
+						</input>
+					</repeat>
+				</group>
+				<trigger ref="/data/outer_repeat/outer_end">
+					<label ref="jr:itext('question31-label')"/>
+				</trigger>
+			</repeat>
+		</group>
+		<input ref="/data/num_repeats">
+			<label ref="jr:itext('outer_repeat/question1-label')"/>
+		</input>
+		<trigger ref="/data/label_num_repeats" appearance="minimal">
+			<label ref="jr:itext('label_num_repeats-label')"/>
+		</trigger>
+		<group>
+			<label ref="jr:itext('num_repeat-label')"/>
+			<repeat jr:count="/data/num_repeats" jr:noAddRemove="true()" nodeset="/data/num_repeat">
+				<input ref="/data/num_repeat/repeats">
+					<label ref="jr:itext('question2/repeats-label')"/>
+				</input>
+			</repeat>
+		</group>
+	</h:body>
+</h:html>
```

Comment:
would be nice to make these files smaller/more concise so it's easier to tell what exactly they're testing